drivers/storage/ide/pciide/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 
 add_library(pciide MODULE pciide.c pciide.rc)
 set_module_type(pciide kernelmodedriver)
+if(MSVC)
+    target_link_options(pciide PRIVATE /SECTION:.rsrc,!D)
+else()
+    # Not implemented
```

Comment:
```suggestion
    add_custom_command(
        TARGET pciide POST_BUILD
        COMMAND objcopy --set-section-flags .rsrc=readonly $<TARGET_FILE:pciide>
        VERBATIM)
```
(Note: if "objcopy" doesn't work, try using `${CMAKE_OBJCOPY}` instead.)

Reply:
objcopy is broken
![result](https://user-images.githubusercontent.com/37072976/114218872-cdf7b900-998b-11eb-8024-a52bf314646f.png)

Reply:
huh yeah indeed definitively. It does much more changes (and sometimes harmful ones) than intended. Let's not use it then.